style.css

```diff
@@ -37,6 +50,46 @@
   display: flex;
 }
 
+.content button a:hover {
+  color: #fff;
+}
+
+.about-myself .about-content button a {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  transition: all 1s;
+}
```

Comment:
- [ ] Kindly use flexbox to align the content in the about section to match all the project requirements 👇🏽 

> Make sure the content of the "About myself" section is aligned with flexbox.